### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by elduddz**
* please review Naming convention on module name

[ ] Testing required

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+}
+
+variable "newbits" {
+  default = 4
+}
+
+variable "service_endpoints" {  
+}
+
+variable "tags" {
```
Comment (MarcDenman) [inline variables.tf]: Does tags need to be required? Can we default to empty if not please.

File: variables.tf [hunk 2]
```diff
@@ -1,5 +1,22 @@
-variable "name" {
-  type = string
-  default = "test"
+variable "prefix" {  
 }
 
+variable "resource_group_name" {
```
Comment (MarcDenman) [inline variables.tf]: Can you add a description on these variables as well please

File: main.tf [hunk 3]
```diff
@@ -1,5 +1,39 @@
-resource "azurerm_resource_group" "rg" {
-    name = "${var.name}-RG"  
-    location = "UK South"
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscription" "current" {}
```
Comment (MarcDenman) [inline main.tf]: Are these two actually needed?

File: README.md [hunk 4]
```diff
@@ -1 +1,46 @@
...
+## Useage
+
+resource "azurerm_resource_group" "gg" {
+  name = "resourcegroup-rg"
+  location = "UKSouth"
+}
+
+module "setup" {
+  source                        = "github.com/ukho/tfmodule-azure-vnet-with-nsg?ref=1.0.0"
+  prefix                        = "[Put in front of vnet and nsg]"
```
[the remark this answers is not in the document]
Comment (elduddz) [reply README.md]: yeah...

File: variables.tf (hunk 1 above)
Comment (elduddz) [reply variables.tf]: yes, it is linked on the resource, just not set as anything.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (elduddz) [reply README.md]: done

File: variables.tf (hunk 1 above)
Comment (elduddz) [reply variables.tf]: fixed; will obviously break, if the rg doesnt have the tags ;)

File: variables.tf (hunk 2 above)
Comment (elduddz) [reply variables.tf]: description added

File: main.tf (hunk 3 above)
Comment (elduddz) [reply main.tf]: not sure, I thought they had a job... not sure now.

Comment (elduddz) [reply main.tf]: I removed it...

**Pull request merged**